Domain Connect Spec Draft.adoc

```diff
@@ -1067,7 +1147,7 @@ token.
 
 |*Redirect URI*
 |redirect_uri
-|(optional) This is required if a redirect_uri was
+|(OPTIONAL) This is required if a redirect_uri was

```

Comment:
Perhaps all the tables should have RFC2119 colum that tells: REQUIRED, RECOMMENDED, or OPTIONAL.